content/v2.markdown

```diff
@@ -57,3 +57,120 @@ curl -H 'X-DNSimple-Domain-Token: DOMAIN-TOKEN' \
+        "address2": "",
+        "city": "Miami",
+        "state_province": "FL",
+        "postal_code": "33143",
+        "country": "US",
+        "created_at": "2015-05-14T10:05:46.618Z",
+        "updated_at": "2015-05-14T10:05:46.618Z"
+    }
+}
+```
+
+## Errors
+
+Error responses will be returned as a JSON object with at least a `message` key. For example, failure to authenticate would result in a 401 response with a body like this:
+
+```
+{"message":"Authentication failed"}
+```
+
+## Pagination
+
+Any API endpoint that returns a list of items requires pagination. By default we will return 30 records from any listing endpoint.
+
+If an API endpoint returns a list of items, then it will include a `pagination` object that contains pagination information.
+
+The `pagination` object contains the following details:
+
+- `current_page`: The page currently returned (default: 1)
+- `per_page`: The number of entries returned per page (default: 30)
+- `total_entries`: The total number of entries available in the entire collection
+- `total_pages`: The total number of pages available given the current `per_page` value
+
+For example:
+
+```
+{
+    "data": [
+        {
+            "id": 1,
+            "account_id": 1,
+            "label": "",
+            "first_name": "Jane",
+            "last_name": "Smith",
+            "job_title": "",
+            "organization_name": "",
+            "email_address": "janesmith@gmail.com",
+            "phone": "+1 1234567890",
+            "fax": "",
+            "address1": "111 SW 1st Street",
+            "address2": "",
+            "city": "Miami",
+            "state_province": "FL",
+            "postal_code": "33143",
+            "country": "US",
+            "created_at": "2015-05-14T10:05:46.618Z",
+            "updated_at": "2015-05-14T10:05:46.618Z"
+        },
+        {
+            "id": 3,
+            "account_id": 1,
+            "label": "",
+            "first_name": "John",
+            "last_name": "Smith",
+            "job_title": "",
+            "organization_name": "",
+            "email_address": "john.smith@example.com",
+            "phone": "+1 321 4567878",
+            "fax": "",
+            "address1": "111 SW 1st Str",
+            "address2": "",
+            "city": "Miami",
+            "state_province": "FL",
+            "postal_code": "32111",
+            "country": "US",
+            "created_at": "2015-10-07T15:00:14.666Z",
+            "updated_at": "2015-10-07T15:00:14.666Z"
+        }
+    ],
+    "pagination": {
+        "current_page": 1,
+        "per_page": 30,
+        "total_entries": 2,
+        "total_pages": 1
+    }
+}
+```
+
+You may include the following arguments as query parameters when you call an API endpoint that supports pagination:
+
+- `page`: The page to return (default: 1)
+- `per_page`: The number of entries to return per page (default: 30)
+
+For example:
+
+    https://api.dnsimple.com/v2/domains?per_page=100&page=2
```

Comment:
This URL is incorrect. It should be

    https://api.dnsimple.com/v2/:account/domains?per_page=100&page=2

I'm not sure if it's better to simply use

    https://api.dnsimple.com/v2/account/domains?per_page=100&page=2

or a random value such as

    https://api.dnsimple.com/v2/1/domains?per_page=100&page=2